src/PlaywrightSharp.Chromium/PlaywrightSharp.Chromium.csproj

```diff
@@ -1,30 +1,31 @@
-        <LangVersion>8.0</LangVersion>
-        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-        <DocumentationFile>bin\$(Configuration)\netstandard2.0\PlaywrightSharp.Chromium.xml</DocumentationFile>
-        <RunWithWarnings>true</RunWithWarnings>
-        <StyleCopTreatErrorsAsWarnings>false</StyleCopTreatErrorsAsWarnings>
-        <CodeAnalysisRuleSet>../PlaywrightSharp.ruleset</CodeAnalysisRuleSet>
-    </PropertyGroup>
-    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-        <NoWarn>1701;1702;CS0067</NoWarn>
-    </PropertyGroup>
-    <ItemGroup>
-        <ProjectReference Include="..\PlaywrightSharp\PlaywrightSharp.csproj" />
-        <ProjectReference Include="..\PlaywrightSharp.Abstractions\PlaywrightSharp.Abstractions.csproj" />
-    </ItemGroup>
-    <Import Project="../Common/Dependencies.props" />
-    <ItemGroup>
-        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
-            <_Parameter1>PlaywrightSharp.Tests</_Parameter1>
-        </AssemblyAttribute>
-    </ItemGroup>
-    <ItemGroup>
-      <PackageReference Include="Macross.Json.Extensions" Version="1.2.1" />
-      <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
-    </ItemGroup>
-</Project>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+        <ReleaseVersion>0.0.0</ReleaseVersion>
+        <LangVersion>8.0</LangVersion>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <DocumentationFile>bin\$(Configuration)\netstandard2.0\PlaywrightSharp.Chromium.xml</DocumentationFile>
+        <RunWithWarnings>true</RunWithWarnings>
+        <StyleCopTreatErrorsAsWarnings>false</StyleCopTreatErrorsAsWarnings>
+        <CodeAnalysisRuleSet>../PlaywrightSharp.ruleset</CodeAnalysisRuleSet>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+        <NoWarn>1701;1702;CS0067</NoWarn>
+    </PropertyGroup>
+    <ItemGroup>
+        <ProjectReference Include="..\PlaywrightSharp\PlaywrightSharp.csproj" />
+        <ProjectReference Include="..\PlaywrightSharp.Abstractions\PlaywrightSharp.Abstractions.csproj" />
+    </ItemGroup>
+    <Import Project="../Common/Dependencies.props" />
+    <Import Project="../Common/SignAssembly.props" />
+    <ItemGroup>
+        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+            <_Parameter1>PlaywrightSharp.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100059a04ca5ca77c9b4eb2addd1afe3f8464b20ee6aefe73b8c23c0e6ca278d1a378b33382e7e18d4aa8300dd22d81f146e528d88368f73a288e5b8157da9710fe6f9fa9911fb786193f983408c5ebae0b1ba5d1d00111af2816f5db55871db03d7536f4a7a6c5152d630c1e1886b1a0fb68ba5e7f64a7f24ac372090889be2ffb</_Parameter1>
```

Comment:
can we have the `PublicKey` as a constant somewhere

Reply:
I love these push backs, thank you!